### ISSUE TITLE
inference script starts producing bullshit at low temperatures

### ISSUE THREAD

**Issue by alimadelshin** [turboderp/exllamav2#50]
inference script starts producing bullshit at low temperatures and the generation speed drops from 21 to ~1 token per second

**Comment by turboderp** [comment]
Thanks, I missed that. Should be fixed now.

**Comment by Nikita-Sherstnev** [comment]
Now at zero temperature model produces '每每每每每' symbols

**Comment by turboderp** [comment]
Yes, zero temperature would be undefined behavior, since the logits are divided by the temperature before the softmax. But I guess the expected behavior is greedy sampling?
